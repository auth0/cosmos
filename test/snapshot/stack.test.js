import React from 'react'
import renderer from 'react-test-renderer'
import { Stack } from '@auth0/cosmos'
import 'jest-styled-components'

describe('Stack', () => {
  it('renders with automation attribute', () => {
    const tree = renderer
      .create(
        <Stack>
          <div>element</div>
        </Stack>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
