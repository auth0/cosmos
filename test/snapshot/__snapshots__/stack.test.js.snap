// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stack renders with automation attribute 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: nowrap;
  -ms-flex-wrap: nowrap;
  flex-wrap: nowrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c0 > * {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  margin-right: 8px;
}

.c0 > *:last-child {
  margin-right: 0;
}

.c1 {
  -webkit-flex-basis: 0%;
  -ms-flex-preferred-size: 0%;
  flex-basis: 0%;
}

<div
  className="c0"
  data-cosmos-key="stack"
>
  <div
    className="c1"
    data-cosmos-key="stack.item"
    width={0}
  >
    <div>
      element
    </div>
  </div>
</div>
`;
